livekit-server/values.yaml

```diff
@@ -43,6 +47,15 @@ livekit:
   # room: <default settings for new rooms>
   # region: <set for multi-region deploys>
 
+keyFile:
```

Comment:
nit: would it be more clear to call it `storeKeysInSecret` ? It's not clear that by default enabling this field would cause a new secret to be created.

Reply:
Done!